examples/normalizer/Normalize.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+import sys
+import csv
+from trigger.cmds import ReactorlessCommando
+from twisted.python import log
+from twisted.internet import reactor
+from Router import Router
+import jsonpickle
```

Comment:
Where does `jsonpickle` come from? It seems to be 3rd party? If so, can you please make sure that it's mentioned in `README.rst` that this is required for the example to work?